ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,36 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.fft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        shape=(2, 2),
+        array_api_dtypes=True
+    ),
+    s=st.tuples(
```

Comment:
Because input array is not limited to `(2, 2)`, please try to generate something more general:
> Shape (length of each transformed axis) of the output (s[0] refers to axis 0, s[1] to axis 1, etc.).

It can even be `None`, so please try to include `None` as well.

Reply:
So I'm not sure I'm able to include '''None''' in this field. It throws a '''TypeError: unsupported operand type(s) for *=: 'int' and 'NoneType'''
Is there a better way to include None types here?

Reply:
So I'm not sure I'm able to include `None` in this field. It throws a `TypeError: unsupported operand type(s) for *=: 'int' and 'NoneType'`
Is there a better way to include None types here?

Reply:
This seems to work for me:
```python
s=st.sampled_from([None, st.tuples(
        st.integers(min_value=2, max_value=5),
        st.integers(min_value=2, max_value=5)
)])
```

The test seems to be failing if `s=None` though, can you try to fix them?